ipyopenlayers/_version.py

```diff
@@ -4,5 +4,5 @@
 # Copyright (c) QuantStack.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 2, 1)
+version_info = (0, 3, '0-rc1')
```

Comment:
I will do this during the release

```suggestion
version_info = (0, 2, 1)
```